resources/serverless/charts/docker-registry/values.yaml

```diff
@@ -116,12 +116,49 @@ configData: # example: https://github.com/docker/distribution/blob/master/cmd/re
       interval: 10s
       threshold: 3
 
-securityContext:
-  enabled: true
-  runAsUser: 1000
-  fsGroup: 1000
-  privileged: false
-  allowPrivilegeEscalation: false
+containers:
+  securityContext:
+    runAsUser: 1000
+    fsGroup: 1000
```

Comment:
I see that `fsGroup` is not visible when I deploy the chart on the cluster and get deployment using the kubectl. Is it ok?

```
      securityContext:
          allowPrivilegeEscalation: false
          privileged: false
          procMount: Default
          readOnlyRootFilesystem: true
          runAsUser: 1000
```

Reply:
I think it's not supported under containers spec. It was moved from the original chart. I will remove it.